### PR TITLE
Feat/book model

### PR DESCRIPTION
**Pull request by RobertDataWow**
Add model only

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Book < ApplicationRecord
+  has_many :reviews
```
Comment (kanatsanan6) [inline app/models/book.rb]: we can specify destroy behavior by adding `dependent` options
ref: https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent

File: db/migrate/20230822090606_create_books.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+      t.string :name
```
Comment (kanatsanan6) [inline db/migrate/20230822090606_create_books.rb]: we can add constraints directly to a database by adding column modifiers

https://guides.rubyonrails.org/active_record_migrations.html#column-modifiers

**Review by kanatsanan6 — APPROVED** [review]

**Review by WatcharitNew — APPROVED** [review]
As Toon's comment

**Comment by RobertDataWow** [comment]
@kanatsanan6 @WatcharitNew 
- [x] we can specify destroy behavior by adding `dependent` options
- [x] we can add constraints directly to a database by adding column modifiers

**Pull request merged**